README.md

```diff
@@ -35,48 +35,60 @@
+
+🛠️ **[Tools](https://docs.haystack.deepset.ai/v1.15/docs/agent#tools):** You can think of a Tool as an expert, that is able to do something really well. Such as a calculator, good at mathematics. Or a [WebRetriever](https://docs.haystack.deepset.ai/v1.15/docs/agent#web-tools), good at retrieving pages from the internet. A Node or pipeline in Haystack can also be used as a Tool. A Tool is a component that is used by an Agent, to resolve complex queries.
+
+**[DocumentStores](https://docs.haystack.deepset.ai/docs/document_store):** A DocumentStore is database where you store your text data for Haystack to access. Haystack DocumentStores are available with ElasticSearch, Opensearch, Weaviate, Pinecone, FAISS and more. For a full list of available DocumentStore, check out our [documentation](https://docs.haystack.deepset.ai/docs/document_store).
 
 ## What to Build with Haystack
 
-- **Ask questions in natural language** and find granular answers in your documents.
-- Perform **semantic search** and retrieve documents according to meaning, not keywords.
-- Use **off-the-shelf models** or **fine-tune** them to your domain.
-- Use **user feedback** to evaluate, benchmark, and continuously improve your live models.
-- Leverage existing **knowledge bases** and better handle the long tail of queries that **chatbots** receive.
-- **Automate processes** by automatically applying a list of questions to new documents and using the extracted answers.
-
-## Core Features
-
-- **Latest models**: Utilize all latest transformer-based models (for example, BERT, RoBERTa, MiniLM) for extractive QA, generative QA, and document retrieval.
-- **Modular**: Multiple choices to fit your tech stack and use case. Pick your favorite database, file converter, or modeling framework.
-- **Pipelines**: Use the Node and Pipeline design of Haystack to route queries to only the relevant components.
-- **Open**: 100% compatible with Hugging Face's model hub. Tight interfaces to other frameworks (for example, transformers, FARM, sentence-transformers).
-- **Scalable**: Scale to millions of docs using retrievers, production-ready backends like Elasticsearch / FAISS, and a fastAPI REST API.
-- **End-to-End**: All tooling in one place: file conversion, cleaning, splitting, training, eval, inference, labeling, and more.
-- **Developer friendly**: Easy to debug, extend, and modify.
-- **Customizable**: Fine-tune models to your domain or implement your custom DocumentStore.
-- **Continuous Learning**: Collect new training data from user feedback in production & improve your models continuously.
+-   Question Answering **in natural language** to find granular answers in your documents.
```

Comment:
To be consistent with other bullet points here, I'd start with a verb: Build systems for Question Answering.." or "Perform Question Answering.."